principles/devops.md

```diff
@@ -5,5 +5,52 @@ title:  "DevOps Principles"
+
+Adhereing to DevOps provides the service with the following benefits:
+
+- **Speed and quality.** DevOps speeds up our releases using continuous delivery, encouraging faster feedback, and allowing our developers to fix bugs in the system in the early stages. We can focus on the quality of the services we build and automate processes.
+
+- **Apprenticeship benefits.** With DevOps, we can react to change requests from users faster, adding new features and updating existing. As a result, value-delivery rates increase.
+
+- **Better internal culture.** DevOps principles lead to better communication between our team members, and increased productivity and agility. We are more productive and cross-skilled.
+
+There are many blogs, books and courses on DevOps, and the CAMS model is often written about. The CAMS model was invented by Damon Edwards and John Willis, authors of the famous Podcast, DevOps Cafe. CAMS stands for Culture, Automation, Measurement and Sharing. Using the CAMS model, we can break down our core principles and provide an insight into how we deliver and continuously improve on those principles.
+
+## Culture
+
+The purpose of DevOps in the Apprenticeship service is to bring unity among the teams so that they all work as a single entity to achieve a goal or rollout new features and services. Working together creates a positive attitude and in turn improves the overall environment for the teams to work in. This enables continuous learning and sharing of skills. All this leads to efficient and innovative development and delivery. These are some of the cultural advantages DevOps provides for the service:
+
+- **Agile ways of working.** Across the service we perform daily collaboration through Agile ceremonies; sprint planning, daily standup, sprint reviews and retrospectives. Support and trust are key to our ways of working which helps enable our continuous delivery. There are also other ceremonies that bring all of the teams together such as the Scrum of Scrums, Clan Leads meeting, Architecture board, Quality Guild and a regular service wide standup.
+
+- **Show and tell.** These are regular meetings that allow teams to celebrate their work and talk about what they’ve learned. It's a chance to bring teams together, share success and improve on collaboration. Progress is highlighted along with question and feedback sessions. Transparency is encouraged to glimpse 'behind the curtain' of the products in development and to highlight where dependencies may exist between teams. A great show and tell session is one that is performed on a regular basis to ensure continuous feedback.
+
+## Automation
+
+This is the most vital factor of the DevOps culture. The purpose of automation is to build a system such that it eliminates various human errors and can fix errors and repeating the successful process again without any sort of intervention or delays. Automation greatly improves the workflow and productivity of the service. 
```

Comment:
bit of an awkward sentence: `...human errors and can fix errors and repeating the successful process again...`

Reply:
Updated 👍